mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/KitResult.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *  * dataloader
+ *  * Copyright © $DateInfo.year SolarMC Developers
+ *  *
+ *  * dataloader is free software: you can redistribute it and/or modify
+ *  * it under the terms of the GNU Affero General Public License as
+ *  * published by the Free Software Foundation, either version 3 of the
+ *  * License, or (at your option) any later version.
+ *  *
+ *  * dataloader is distributed in the hope that it will be useful,
+ *  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  * GNU Affero General Public License for more details.
+ *  *
+ *  * You should have received a copy of the GNU Affero General Public License
+ *  * along with dataloader. If not, see <https://www.gnu.org/licenses/>
+ *  * and navigate to version 3 of the GNU Affero General Public License.
+ *
+ */
+
+package gg.solarmc.loader.kitpvp;
+
+public class KitResult {
+    //if this were c#, i'd have to do a constructor and a private bool Success {get;set;} but noooooooooooooooooooooooooo
+
+    private final boolean success;
+
+    KitResult(boolean success) {
+        this.success = success;
+    }
+
+    public boolean isSuccess() {
```

Comment:
The method name is misleading. Success/failure is not the right choice of terms. More like `isChanged`

Also, I think it would be clearer to name this class `KitOwnershipResult`

Reply:
Fixed